Core/Graphics/Source/Graphics.cpp

```diff
@@ -64,12 +64,15 @@ namespace Babylon
 
     void Graphics::Impl::Resize(size_t width, size_t height)
     {
-        std::scoped_lock lock{m_bgfxState.Mutex};
+        m_resolution.width = width;
```

Comment:
Using a lock here and changing this from JS via `OnBeforeRender` caused an exception. Though we should probably figure out why instead of removing the lock

Reply:
Re-added the lock.